File: Programmers/Inho/Level01/없는 숫자 더하기.cs
using System;

public class Solution
{
    public int solution(int[] numbers)
    {
        int answer = -1;
        Array.Sort(numbers);
        int count = 0;
        foreach (int a in numbers)
        {
            answer += a;
        }
        return 44 - answer;
    }
}

File: Programmers/Inho/C++/Level01/없는 숫자 더하기.cpp
#include <string>
#include <vector>
#include <algorithm>
#include <iostream>

using namespace std;

int solution(vector<int> numbers) {
    int answer = 0;
    sort(numbers.begin(), numbers.end());
    // i가 0~9 숫자 커서 , j가 numbers 커서
    for(int i = 0, j = 0; i<10;i++, j++){
        if(i!=numbers[j]){
            j--;
            answer+=i;
        }
    }
    return answer;
}